### ISSUE TITLE
Submandibular lymph node enlargement

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6212]
**Preferred term label:**
Submandibular lymph node enlargement

**Synonyms**
Submandibular lymphadenopathy
**Definition (free text, please give PubMed ID)**
Increased size of the lymph nodes that are located beneath the mandible (lower jaw).

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
many.
Here is one
10/24, Cherubism, OMIM:118400, PMID: 16053841

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
